components/tab/content/src/index.js

```diff
@@ -0,0 +1,90 @@
+import React, {Component, PropTypes} from 'react'
+import TabsBasic from '@schibstedspain/sui-tab-basic'
+import cx from 'classnames'
+
+class TabContent extends Component {
+  state = {
+    activeTab: this.props.activeTab
+  }
+
+  _handleClick = (activeTab) => {
+    this.setState({ activeTab })
+  }
+
+  _renderContent ({ activeTab, tabsContent }) {
+    const { renderInactiveContent } = this.props
+    return tabsContent.reduce((contentToRender, content, indexTab) => {
+      const isActive = indexTab === activeTab
+      if (isActive || renderInactiveContent) {
+        const className = cx('sui-TabContent-content', {
+          'is-active': isActive
+        })
+
+        contentToRender.push((
+          <section className={className} key={indexTab}>
+            {content}
+          </section>
+       ))
+      }
+      return contentToRender
+    }, [])
+  }
+
+  shouldComponentUpdate (nextProps, nextState) {
+    // re-render the component only if the user is changing the activated tab
+    // we might want to make the content of the tabs dynamic as well and we may change this
+    return nextState.activeTab !== this.state.activeTab
+  }
+
+  render () {
+    const { panels } = this.props
+    const { activeTab } = this.state
+
+    const tabsList = panels.map(panel => panel.title)
+    const tabsContent = panels.map(panel => panel.content)
+
+    return (
+      <div className='sui-TabContent'>
+        <TabsBasic
+          activeTab={activeTab}
+          handleClick={this._handleClick}
+          tabsList={tabsList}
+        />
+        {this._renderContent({ activeTab, tabsContent })}
+      </div>
+    )
+  }
+}
+
+TabContent.displayName = 'TabContent'
+
+TabContent.defaultProps = {
+  activeTab: 0,
+  renderInactiveContent: true
```

Comment:
For SEO reasons, we might want to render the HTML of the hidden tabs but usually we would want to avoid that and don't render the invisble content at all. This option offers you both possibilities.